app/services/hyrax/solr_query_service.rb

```diff
@@ -17,6 +20,21 @@ def get
       solr_service.get(build)
     end
 
+    ##
+    # @return [Array<String>] ids of documents matching the current query
+    def get_ids # rubocop:disable Naming/AccessorMethodName
+      results = get
+      results['response']['docs'].map { |doc| doc['id'] }
+    end
+
+    ##
+    # @return [Array<Valkyrie::Resource|ActiveFedora::Base>] objects matching the current query
+    def get_objects(use_valkyrie: Hyrax.config.use_valkyrie?)
+      ids = get_ids
+      return ids.map { |id| ActiveFedora::Base.find(id) }.to_a unless use_valkyrie
+      query_service.find_many_by_ids(ids: ids).to_a
```

Comment:
is the `#to_a` desirable, or could this return an `Enumerator`?

it seems like avoiding loading data that won't be used is attractive.

Reply:
That makes sense. I'll remove it and see if everything continues to work.  I used `to_a` because the deprecated method used it.  I can't remember the original reason for doing using `to_a`.